src/components/NcBreadcrumbs/NcBreadcrumbs.vue

```diff
@@ -653,12 +653,11 @@ export default {
 
 	nav {
 		flex-shrink: 1;
-		max-width: 100%;
 		/**
 		 * This value is given by the min-width of the last crumb (100px) plus
 		 * two times the width of a crumb with an icon (first crumb and hidden crumbs actions).
 		 */
-		min-width: 228px;
+		min-width: 226px;
```

Comment:
```suggestion
		min-width: 0;
```

Otherwise anything after gets pushed to the right
![Peek 18-10-2023 20-11](https://github.com/nextcloud-libraries/nextcloud-vue/assets/14975046/9ff1f133-2f60-4c7d-97f5-9421ba124dae)

Reply:
Thanks, done